test/configuration_test.rb

```diff
@@ -187,6 +187,17 @@ def hash
     end
   end
 
+  describe '#format_mime_types' do
+    before do
+      @configuration.format custom: 'custom/format'
+    end
+
+    it 'returns all known format MIME types' do
```

Comment:
This description should probably read "return all configured format MIME types".